libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo/ApolloCall.kt

```diff
@@ -147,34 +147,6 @@ class ApolloCall<D : Operation.Data> internal constructor(
     return apolloClient.executeAsFlowInternal(requestBuilder.build(), false)
   }
 
-  /**
-   * A version of [execute] that restores 3.x behaviour:
-   * - throw on fetch errors.
-   * - make `CacheFirst`, `NetworkFirst` and `CacheAndNetwork` policies ignore fetch errors.
-   * - throw ApolloComposite exception if needed.
-   */
-  @Deprecated("Use toFlow() and handle ApolloResponse.exception instead")
-  @ApolloDeprecatedSince(ApolloDeprecatedSince.Version.v4_0_0)
-  fun toFlowV3(): Flow<ApolloResponse<D>> {
```

Comment:
The V3 helpers are also not going through `ERROR` because I think it's relatively clear that they were a temporary solution,